supervisely/api/video/video_annotation_api.py

```diff
@@ -254,18 +252,28 @@ async def download_async(
         video_id: int,
         video_info=None,
         semaphore: Optional[asyncio.Semaphore] = None,
+        force_metadata_for_links: bool = True,
+        integer_coords: bool = True,
+        progress_cb: Optional[Union[tqdm, Callable]] = None,
     ) -> Dict:
         """
         Download information about VideoAnnotation by video ID from API asynchronously.
 
         :param video_id: Video ID in Supervisely.
         :type video_id: int
-        :param video_info: VideoInfo object. Use it to avoid additional request to the server.
+        :param video_info: Does not affect the result, but is left for compatibility with the method signature.
         :type video_info: VideoInfo, optional
         :param semaphore: Semaphore to limit the number of parallel downloads.
         :type semaphore: asyncio.Semaphore, optional
+        :param force_metadata_for_links: Force metadata for links.
```

Comment:
not clear descriptions for `force_metadata_for_links` and `integer_coords` params.
(the same in the `download_bulk_async` method)